app/scripts/metamask-controller.js

```diff
@@ -1664,6 +1697,10 @@ module.exports = class MetamaskController extends EventEmitter {
     await this.preferencesController.removeFromFrequentRpcList(rpcTarget)
   }
 
+  async initializeThreeBox (address) {
+    await this.threeBoxController.new3Box(address, true)
```

Comment:
Maybe this should always grab the first address, rather than accepting an address as an input. Otherwise it could end up being initialized with different accounts depending on where in the flow it became initialized.

Reply:
done in 40a3c5879